library/ssl_msg.c

```diff
@@ -1,7 +1,8 @@
 /*
- *  SSLv3/TLSv1 shared functions
+ *  Generic SSL/TLS messaging layer functions
```

Comment:
In commit “Move messaging layer functionality to ssl_msg.c”, the commit description says

> the newly created (and thus far empty) ssl_msg.c.

But at this point ssl_msg.c is not empty: it's a copy of ssl_tls.c. Please fix the commit description.

Reply:
Fixed.